addons/sales_team/views/crm_team_views.xml

```diff
@@ -79,7 +79,7 @@
                                 </kanban>
                             </field>
                             <field name="crm_team_member_ids" mode="kanban"
-                                class="w-100"
+                                class="w-100 o_crm_team_member_kanban"
```

Comment:
Hello @amdi-odoo LGTM, but I can't find why this is only targeting master. (In practice, I see this works)

Did you find the change that made this actually needed? Because I see gauge field has been there for a while, and if I am not mistaken, gauge has been brought to owl before saas~16.1. 

So in short, and for ref. : what has made this forgotten class useful now? Any commit ref introducing the issue, that I missed?

Thanks for your insight :)

Reply:
Ok so here is the field displaying the kanban :
```xml
  <field name="crm_team_member_ids" mode="kanban"
      class="w-100"
      attrs="{'invisible': ['|', ('is_membership_multi', '=', True), ('is_membership_multi', '=', False)]}"
      context="{
          'kanban_view_ref': 'sales_team.crm_team_member_view_kanban_from_team',
          'form_view_ref': 'sales_team.crm_team_member_view_form_from_team',
          'tree_view_ref': 'sales_team.crm_team_member_view_tree_from_team',
          'default_crm_team_id': active_id,
      }"/>
```
It's in kanban mode so using the `sales_team.crm_team_member_view_kanban_from_team` view which inherit from the `crm_team_member_view_kanban` one having a kanban with the class `o_crm_team_member_kanban` :
```xml
            <kanban quick_create="false"
                group_create="0"
                records_draggable="0"
                sample="1"
                default_group_by="crm_team_id"
                class="o_crm_team_member_kanban">
```
This is the case both in 16.1 and master.

--------------------------------------------------------------

Now : 
In 16.1, the kanban mode field inherit the `o_crm_team_member_kanban` class from its view and so this css rule is correctly applied (the display is done correctly)  : 
```css
.o_kanban_renderer.o_crm_team_member_kanban {
    --KanbanRecord-width: 350px;
}
 ``` 

 But for a certain reason in master, this `o_crm_team_member_kanban` class isn't inherited from the view. So the kanban mode field don't have the class on. (which makes the display look bad).

 Moreover, from 16.1 to master, the css rule has been redefined as :
 ```css
.o_crm_team_member_kanban .o_kanban_renderer {
    --KanbanRecord-width: 350px;
    }
```
(from this commit https://github.com/acsone/odoo/commit/2bc62279baec3f0ed0d637bafdadcfdd1a7348bd#diff-f514c37124b2cac1dd49c44b8578e992cecbc0e67e6d255dd6d7783f98d45737 in the file `addons/sales_team/static/src/scss/crm_team_views.scss` )

So 1 :
- I don't get why the `o_crm_team_member_kanban` css class isn't inherited from the view to the kanban mode field anymore.

and 2 :
- To fix the display we need to set the class on the kanban mode field again, so I just added the class directly on it (which I think is maybe not the right thing to do ? :thinking: )

Well I'm kind of stuck right now because I don't understand why the class isn't on the kanban mode field anymore. Will investigate a bit more :slightly_smiling_face: (Sorry the explanations are quite long)

Reply:
Thank you for the investigations! What a story ^^ I would still do the same change as you did, but indeed, it seems there is anguille sous roche somewhere. Most probably in the redefinition.

This could also be of interest https://github.com/odoo/odoo/pull/113014 (last week)

It looks like a side-effect of several kanban view modifications. I'm having a closer look but we should probably not spend too much time on this. :)

Reply:
Oh! I think I got it! 
https://github.com/odoo/odoo/pull/113014/commits/c857473150e4b411f58cae73d098ae9269bd44eb#diff-823da6a5974aafef6ec7d54b4d410ccd8dc54e1892aee57c3bb62ff8a0414826 
```xml
{{ props.archInfo.className or '' }}
```
This disappeared from the kanban renderer xml so I think it doesn't apply the class anymore. :thinking: 
If you think my solution is good then we could go with it?

Reply:
Yes, as it is a field, I think we can do that :)

Reply:
@amdi-odoo You can always mention those commits in the CM / PRM, for reference. I've greened, thanks for the researches :)

Reply:
Crazy story indeed :smile: 

Did you ask the JS team if that was an intended side-effect from their changes?
Because it seems like the commit you mentioned was working on the js_class feature, and had nothing to do with class attribute inheritance :thinking: 

Would be interesting to have their opinion, maybe something was unintentionally broken and we could fix it globally instead of having to fix it view by view.